### ISSUE TITLE
TypeError: Cannot convert kivy.properties.NumericPropertyStorage to kivy.properties.BoundedNumericPropertyStorage

### ISSUE THREAD

**Issue by olek5andr** [kivymd/KivyMD#995]
Python == 3.7.6
kivy == 2.1.0.dev0
KivyMD == 0.104.2.dev0

App crashes after updating from kivy == 2.0.0 to kivy == 2.1.0.dev0.
On kivy==2.0.0 the app runs PERFECTLY on mac, but lags extensively on an Android device. That's the main reason behind updating to kivy master.

The Error message:

```
kivy.lang.builder.BuilderException: Parser: File "<inline>", line 16:
 ...
      14:    size_hint: None, None
      15:    height: dp(20) + lbl_txt.texture_size[1]
 >>   16:    width: lbl_txt.texture_size[0] + dp(24)
      17:
      18:    MDLabel:
 ...
 TypeError: Cannot convert kivy.properties.NumericPropertyStorage to kivy.properties.BoundedNumericPropertyStorage
```

PS. I thought this was an issue with kivy, but I was told it's actually caused by KivyMD - https://github.com/kivy/kivy/issues/7398#issuecomment-800550183

Many thanks in advance!

**Comment by HeaTTheatR** [comment]
@olek5andr For now, you can use `Kivy version 2.0.0`. Changes have not yet been made to `KivyMD` that were made for `kivy.properties` in version `2.0.0`

**Comment by quitegreensky** [comment]
@HeaTTheatR I think we can close this issue? fixed in #996

**Comment by HeaTTheatR** [comment]
@quitegreensky Not yet, because your recent changes do not completely solve the problem. The width of the buttons is cut.
